### DISCUSSION TITLE
how to send data from the client to the server?

### DISCUSSION THREAD

**Discussion by mateusfg7** [vercel/next.js#21077] [Help] [answered]
I have a form with some data, and I want to submit this data to server with _axios_, but when I make the submit request, I got the error:
![image](https://user-images.githubusercontent.com/40613276/104498530-8f37aa00-55ba-11eb-9c04-adf04b554ccc.png)

the same when I use _fetch_ function:
![image](https://user-images.githubusercontent.com/40613276/104498620-b42c1d00-55ba-11eb-8ca0-6326bb80301c.png)

how can I do this?

[Code File](https://github.com/mateusfg7/clothes-store-manager/blob/main/web/src/pages/clothes/new.tsx)

**Comment by lalit2005** [comment, marked as answer]
You could use [serverless functions](https://vercel.com/docs/serverless-functions/introduction) for this if you are using Vercel for hosting
Instead of sending a POST request to `/clothes`, you can send it to `/api/clothes`
